yt_dlp/extractor/idagio.py

```diff
@@ -179,7 +191,7 @@ def _parse_playlist_metadata(self, playlist_info):
 
 
 class IdagioPlaylistIE(IdagioPlaylistBaseIE):
-    _VALID_URL = r'https?://(?:www\.)?app\.idagio\.com/playlists/(?!personal/)(?P<id>[\w-]+)'
+    _VALID_URL = r'https?://(?:www\.)?app\.idagio\.com(?:/[a-z]+)?/playlists/(?!personal/)(?P<id>[\w-]+)'
     _TESTS = [{
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?app\.idagio\.com(?:/[a-z]{2})?/playlists/(?!personal/)(?P<id>[\w-]+)'
    _TESTS = [{
```
again, add `only_matching` test please